test/powershell/Modules/Microsoft.PowerShell.Management/Rename-Item.Tests.ps1

```diff
@@ -31,9 +31,9 @@ Describe "Rename-Item tests" -Tag "CI" {
         $oldSp = "$wdSp/$oldSpBName"
         $newSpName = "[renamed]file.txt"
         $newSp = "$wdSp/``[renamed``]file.txt"
-        In $wdSp -Execute {
+        In $wdSp -execute {
```

Comment:
```suggestion
        In $wdSp -Execute {
```

Reply:
This is correct, but opened upstream PR: https://github.com/pester/Pester/pull/1523